src/select-field.tsx

```diff
@@ -117,7 +118,7 @@ export function SelectField<O extends SelectOption = SelectOption>({
                 noOptionsMessage={noOptionsMessage}
                 isClearable={isClearable}
                 menuPlacement={menuPlacement}
-                placeholder={<span />}
+                placeholder={placeholder || ''}
```

Comment:
sorry about my earlier comment, I now see that you're assigning placeholder to `''` in line 47, so you don't need the `|| ''` here